packages/mobile/src/components/now-playing-drawer/Artwork.tsx

```diff
@@ -20,20 +20,21 @@ const createStyles = (themeColors: ThemeColors) =>
     root: {
       marginLeft: spacing,
       marginRight: spacing,
-      width: dimensions.width - spacing * 2,
-      height: dimensions.width - spacing * 2
+      maxHeight: dimensions.width - spacing * 2,
+      alignSelf: 'center'
```

Comment:
ah that's cool, what dos this do ultimately?

Reply:
It allows the image to scale to its container but never bigger than the width of the screen